docs/topics/proc_interacting-with-the-crud-api.adoc

```diff
@@ -18,7 +18,7 @@ When you have finished creating your application booster, you can interact with
 $ oc get route {app-name}
 ----
 
-[source,option="nowrap",subs="attributes+"]
+[source,bash,option="nowrap",subs="attributes+"]
```

Comment:
This isn't a bash script, so we shouldn't mark it as such. Asciidoctor applies a stylesheet automatically to code block to highlight syntax etc., which wouldn't make sense in this case.

Reply:
Agree with @tradej here.
It's possible we use these code block attributes  it in some places in our docs, but we should not be doing so, because of the syntax highlighting.
Instead, for terminal output examples in plain text, we could drop the `source,bash` attributes, which renders the inside the codeblock in plain monospace font only (no highlights).